src/coord/src/error.rs

```diff
@@ -324,6 +328,9 @@ impl fmt::Display for CoordError {
                 value.quoted(),
                 reason,
             ),
+            CoordError::InvalidReplicaSize { size } => {
+                write!(f, "Size {size} not specified in allowed cluster sizes map")
```

Comment:
In Platform, the user won't know what the "allowed cluster sizes map" is, so recommend an error message like:

```suggestion
                write!(f, "unknown cluster size {size}")
```

Can you also add a hint that displays the known cluster sizes?